packages/itwin/measure-tools/src/tools/MeasureLocationTool.ts

```diff
@@ -223,7 +210,7 @@ MeasureLocationToolModel
       result.snapPoint
     );
     snap.setCurvePrimitive(
-      parseCurve(result.curve),
+      parseCurve(result.primitive),
       undefined,
```

Comment:
why do we still need `parseCurve` ?

Reply:
misinterpreted result.primitive. Thought it was the replacement of result.curve. But I see now that result.primitive is what should be the outcome of parseCurve(result.curve).